components/prism-pascaligo.js

```diff
@@ -0,0 +1,45 @@
+// Pascaligo is a layer 2 smart contract language for the tezos blockchain
+
+Prism.languages.pascaligo = {
+	'comment': /\(\*[\s\S]+?\*\)|\/\/.*/,
+	'string': [
+		{
+			pattern: /(?:'(?:''|[^'\r\n])*')+|\^[a-z]/i,
+			greedy: true
+		},
+		{
+			pattern: /(?:"(?:""|[^"\r\n])*")+|\^[a-z]/i,
+			greedy: true
+		},
+	],
+	'keyword': {
+		pattern: /(^|[^&])\b(?:begin|block|const|else|end|for|from|function|if|is|nil|remove|return|skip|then|type|var|while|with)\b/i,
+		lookbehind: true
+	},
+	'boolean': {
+		pattern: /(^|[^&])\b(?:True|False)\b/i,
+		lookbehind: true
+	},
+    'builtin': {
+        pattern: /(^|[^&])\b(?:int|unit|string|nat|map)\b/i,
```

Comment:
Shouldn't this also include [records](https://ligolang.org/docs/next/language-basics/cheat-sheet/)?
Example:
```
type person is record
  age: int ;
  name: string ;
end
```